### PR TITLE
update scripts for packer build:

### PR DESCRIPTION
**Pull request by PaynePei**
1.install infrasim-compute with source code instead of python library 2.fix some typos

### CODE REVIEW

**Comment by fub2** [comment]
+1

File: packer/scripts/infrasim-compute.sh [hunk 1]
```diff
@@ -3,14 +3,17 @@
...
-apt-get -y install socat ipmitool qemu openipmi python-pip libssl-dev libssh-dev libpython-dev libffi-dev sgabios 
+apt-get install -y python-pip libssl-dev libpython-dev git
 
 pip install setuptools
 pip install --upgrade pip
 sleep 1
 
 # install infrasim-compute
-pip install infrasim-compute
+git clone https://github.com/InfraSIM/infrasim-compute.git
```
Comment (turtle-fly) [inline packer/scripts/infrasim-compute.sh]: Shall we remove code after install?

Comment (PaynePei) [inline packer/scripts/infrasim-compute.sh]: good comment
i planed to remove but @fub2 prefer to keep the source code

Comment (turtle-fly) [inline packer/scripts/infrasim-compute.sh]: keep source code help user to identify the exact commit of his/her environment
remove source code hide all details to common user

I'm OK with both, just confirm

**Comment by turtle-fly** [comment]
:+1:

**Pull request merged**